src/dict/Dictionary.ts

```diff
@@ -163,6 +163,14 @@ export class Dictionary<K extends DictionaryKeyTypes, V> {
             return createCellValue();
         },
 
+        /**
+         * Create standard slice value
+         * @returns DictionaryValue<Slice>
+         */
+        Slice: () => {
+            return createSliceValue();
```

Comment:
Using slices as dictionary values is strictly prohibited cause slice is mutable and this can lead to unexpected behavior

Reply:
Hmm, right. But using slices (containing bits and refs) as dictionary values is normal in smart contract implementations (they are not actually mutable in func). How should we work with such dicts in `ton-core`?